packages/web/src/components/pages/WebsiteResults.tsx

```diff
@@ -84,15 +86,30 @@ export function WebsiteResultsPage() {
       />
     );
   }
+
+  const title = `List of NPM packages that are used on ${hostname} - GradeJS`;
+  const description =
+    `GradeJS has discovered ${packagesStats.total} NPM packages used on ${hostname}` +
+    (packagesStats.vulnerable > 0 ? `, ${packagesStats.vulnerable} are vulnerable` : '') +
+    (packagesStats.outdated > 0 ? `, ${packagesStats.outdated} are outdated` : '');
+
   return (
-    <Website
-      isLoading={isLoading}
-      isPending={isPending}
-      webpages={webpages}
-      packages={packagesFiltered}
-      host={hostname ?? ''}
-      vulnerabilities={vulnerabilities}
-      onFiltersApply={setFilters}
-    />
+    <>
+      <Helmet>
+        <title>{title}</title>
+        <meta name='description' content={description} />
+        <meta property='og:title' content={title} />
+        <meta property='og:description' content={description} />
```

Comment:
`og:url` is missing here as well

Reply:
Still missing

Reply:
It's not. og:url tag is taken from upper scope with proper location.